Java/libraries/recognizers-text-number-with-unit/src/main/java/com/microsoft/recognizers/text/numberwithunit/extractors/BaseMergedUnitExtractor.java

```diff
@@ -173,9 +176,9 @@ private void mergePureNumber(String source, List<ExtractResult> ers) {
             }
         }
 
-        Collections.sort(ers, (Comparator) (xO, yO) -> {
-            ExtractResult x = (ExtractResult) xO;
-            ExtractResult y = (ExtractResult) yO;
+        Collections.sort(ers, (Comparator<ExtractResult>) (xo, yo) -> {
+            ExtractResult x = (ExtractResult) xo;
```

Comment:
Nit: There was nothing wrong with the original names here. Maybe the linting rules can be tweaked a bit?